### ISSUE TITLE
using 3 thread leads to 600 %cpu

### ISSUE THREAD

**Issue by tinybright** [madMAx43v3r/chia-plotter#533]
my cmd
chia_plot -t /p/n23/ -2 /p/n23/ -d /plots/c1/  -r 4 -n -1 -u 64

top -c
5128 root      20   0 7463596   6.4g   1676 S 667.9 10.2   3076:21 chia_plot -t /p/n23/ -2 /p/n23/ -d /plots/c1/

**Comment by madMAx43v3r** [comment]
yes, more threads are created, it's a minimum thread count, not maximum
